device/celestica/x86_64-cel_seastone-r0/plugins/psuutil.py

```diff
@@ -18,33 +20,41 @@ def __init__(self):
         PsuBase.__init__(self)
         # DX010 PSU pin mapping
         self.psu = [
-            {'base':216},           # Reserved
+            {'base': self.get_gpio_base()},
             {'abs':27, 'power':22},
             {'abs':28, 'power':25}
         ]
 
+    def get_gpio_base(self):
+        sys_gpio_dir = "/sys/class/gpio"
+        try:
+            for r in os.listdir(sys_gpio_dir):
+                if "gpiochip" in r:
+                    return int(r[8:],10)
+        except Exception as error:
+            logging.error("Unable to get gpio base")
+
+
     def init_psu_gpio(self, pinnum):
         # export pin, input as default
         gpio_base = self.psu[0]['base']
         export_file = "/sys/class/gpio/export"
-        direction_file = '/sys/class/gpio/gpio' + str(gpio_base+pinnum) + '/direction' 
-
         try:
             with open(export_file, 'w') as fd:
                 fd.write(str(gpio_base+pinnum))
         except Exception as error:
-            logging.error("Unable to export gpio ", pinnum)
+            logging.error("Unable to export gpio ", str(gpio_base+pinnum))
```

Comment:
This is improper syntax. Please change to 

`logging.error("Unable to export gpio " + str(gpio_base+pinnum))`

Reply:
OK, I will change it.